pom.xml

```diff
@@ -261,6 +262,12 @@
                 <version>${project.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>mysql</groupId>
+                <artifactId>mysql-connector-java</artifactId>
+                <version>${mysql.connector.version}</version>
+            </dependency>
+
```

Comment:
Please remove this dependency, we should already define this in dolphinscheduler-bom

Reply:
have done